frontend/src/app/main/ui/ds/controls/utilities/input_field.cljs

```diff
@@ -65,8 +67,10 @@
                                                   :variant-seamless (= variant "seamless")
                                                   :variant-dense (= variant "dense")
                                                   :variant-comfortable (= variant "comfortable")))}
+     (when (some? slot-start)
+       slot-start)
      (when (some? icon)
        [:> icon* {:icon-id icon :class (stl/css :icon) :on-click on-icon-click}])
      [:> "input" props]
-     (when (some? children)
-       [:div {:class (stl/css :input-actions)} children])]))
+     (when (some? slot-end)
```

Comment:
You have removed "children" prop in the definition but in file frontend/src/app/main/ui/ds/controls/input.cljs
This component is passing a children.